scripts/tests/ec2-memory-benchmark/README.md

```diff
@@ -13,6 +13,10 @@ All the other pieces will be downloaded automatically (from the
 either locally, where you're running the script, or on the newly
 created instance.
 
+The script downloads an Ansible config file called `main.yml` which you
+should copy to `main2.yml` and edit to enable "sampledata". This config
```

Comment:
My only question would be if there's a way to handle this without the copy, i.e. we have a version upstream with sample data turned on, or we allow a parameter for sample data so that it can be turned on without having to change the yaml file.

Reply:
@donsizemore and I have been discussing this. I believe the strategy would be to improve the spin up scripts to pass multiple `-g` arguments to ansible and one of those could be "sampledata = enabled" (paraphrasing).